clients/go/apis/v1alpha1/config.go

```diff
@@ -64,6 +64,16 @@ type Config struct {
 	// If fail-close is not chosen, the audit logger will log errors that occur,
 	// and then continue without impeding the application in any way.
 	LogMode string `yaml:"log_mode,omitempty" env:"LOG_MODE,overwrite"`
+
+	// JVSEndpoint is the endpoint where public keys may be retrieved from the JVS.
+	// These will be used to validate JWT tokens that are passed in through the
+	// "justification_token" header.
+	JVSEndpoint string `yaml:"jvs_endpoint,omitempty" env:"JVS_ENDPOINT,overwrite,default=localhost:8080"`
+
+	// EnableJustification enables adding justification information to audit logs. If this is enabled,
+	// all manual calls are expected to pass in a justification in the "justification_token" header.
+	// If omitted, justifications will not be added to logs, even if provided.
+	EnableJustification bool `yaml:"enable_justification,omitempty" env:"ENABLE_JUSTIFICATION,overwrite,default=false"`
```

Comment:
I think this is fine for now, but we should consider an enhancement in the future of making this an enum like `JustificationMode`. I can see customer's wanting any of the following behavior:

- Do nothing with justifications
- If a justification is provided, check the signature and add the result as another field (e.g. `justification_verified: true` or `false`)
- If a justification is provided, check the signature and reject the request if it's invalid. If a justification is not provided, accept the request.
- Always require justification and reject requests without justification or requests where justification validation fails

Reply:
https://github.com/abcxyz/lumberjack/issues/257